espnet/nets/pytorch_backend/transformer/encoder.py

```diff
@@ -129,6 +129,8 @@ def __init__(
                 pos_enc_class(attention_dim, positional_dropout_rate),
```

Comment:
Ditto (Can you avoid adding changes to the main transformer modules?)
But these small changes are acceptable.
Could you please add explanations of why you added such a special input layer?